src/main/java/io/zeebe/monitor/WebSocketConfig.java

```diff
@@ -17,6 +22,11 @@ public void configureMessageBroker(MessageBrokerRegistry config) {
 
   @Override
   public void registerStompEndpoints(StompEndpointRegistry registry) {
-    registry.addEndpoint("/notifications").withSockJS();
+    String[] allowedOriginsUrlArr = this.allowedOriginsUrls.split(";");
```

Comment:
This code crashes when the setting is not present.
For better robustness, please change so that even if the property is not configured, the code will work.

Reply:
Should be fine now